Don't use git to get information. It won't work because we're not building in a
git clone. These placeholders will be replaced by code in the Portfile.
https://github.com/egallesio/STklos/issues/515
--- utils/generate-git-info.orig
+++ utils/generate-git-info
@@ -6,4 +6,4 @@
-git_tag="$(git describe --tags --abbrev=0 2>/dev/null || true)"
-git_branch="$(git rev-parse --abbrev-ref HEAD 2>/dev/null || true)"
-git_commit="$(git rev-parse --short HEAD 2>/dev/null || true)"
-git_modified="$(git diff-index --name-only HEAD 2>/dev/null |  xargs -n 1 printf ' \\"%s\\"')"
+git_tag=stklos-@VERSION@
+git_branch=master
+git_commit=@COMMIT@
+git_modified=
